rule-beans/samples/RuleBeansDemos/src_hd/com/axonivy/ivy/supplements/rule/beans/demos/TaxesDialog/TaxesDialogProcess.mod

```diff
@@ -1,71 +1,56 @@
 [Ivy]
-[>Created: Wed Jun 07 15:24:17 CEST 2017]
-15C82BA28CB7DA31 3.20 #module

```

Comment:
would be much easier to diff if this conversion comes with an extra commit ... and then later do the actual change ... but the commit history looks like someone had git client issues ;)